doc/source/user/configoptions.rst

```diff
@@ -68,6 +68,20 @@ they can be limited to only the current thread with
 For boolean options, the values YES, TRUE or ON can be used to turn the option on;
 NO, FALSE or OFF to turn it off.
 
+How to detect if the passed configuration option is known of GDAL
```

Comment:
Here and a couple of other places in this PR mention "of GDAL", and those should be "to GDAL".

Reply:
thanks, fixed